### DISCUSSION TITLE
How to have blog posts under a Blog section instead on home page?

### DISCUSSION THREAD

**Discussion by kylereddoch** [sylhare/Type-on-Strap#444] [Q&A] [answered]
I would like a link to house the blog posts with the paginator instead of the home page. I have been trying to get that to work but haven't had any luck. Maybe I am missing something. This way the home page could be using for other things.

**Comment by sylhare** [comment, marked as answer]
The home page is via the `home` layout showing the blog, it's done via the `index.html` in the root of the repo.
- Here: https://github.com/sylhare/Type-on-Strap/blob/master/index.html

If you want to display another page, you can have a page for the blog and the home page as something else.
